src/Wallabag/CoreBundle/Entity/Config.php

```diff
@@ -77,34 +88,44 @@ class Config
      *      max = 100000,
      *      maxMessage = "validator.feed_limit_too_high"
      * )
+     *
+     * @Groups({"config_api"})
      */
     private $feedLimit;
 
     /**
      * @var float
      *
      * @ORM\Column(name="reading_speed", type="float", nullable=true)
+     *
+     * @Groups({"config_api"})
      */
     private $readingSpeed;
 
     /**
      * @var string
      *
      * @ORM\Column(name="pocket_consumer_key", type="string", nullable=true)
+     *
+     * @Groups({"config_api"})
```

Comment:
I don't think we need to expose it, it won't be used on client side only on server side